linode/manager
# M3-5593: Convert Search Story to MDX

## Description
Updates the `DebouncedSearchTextField` story from `tsx` to `mdx`

## How to test
- `yarn storybook`
- Go to http://localhost:6006/?path=/docs/components-search and check all the stories
